rvip_roi_parser/package.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>rvip_roi_parser</name>
+  <version>0.0.0</version>
+  <description>The rvip_roi_parser package</description>
+
+  <maintainer email="beyhy@artc.a-star.edu.sg">Bey Hao Yun</maintainer>
+
+  <license>BSD</license>
+
+  <buildtool_depend>catkin</buildtool_depend>
+  <build_depend>roscpp</build_depend>
+  <build_depend>rospy</build_depend>
+  <build_depend>sensor_msgs</build_depend>
+  <build_depend>std_msgs</build_depend>
+  <build_export_depend>roscpp</build_export_depend>
+  <build_export_depend>rospy</build_export_depend>
+  <build_export_depend>sensor_msgs</build_export_depend>
+  <build_export_depend>std_msgs</build_export_depend>
+  <exec_depend>roscpp</exec_depend>
+  <exec_depend>rospy</exec_depend>
+  <exec_depend>sensor_msgs</exec_depend>
+  <exec_depend>std_msgs</exec_depend>
```

Comment:
I believe almost all of these should be `depends` instead.

And `rospy` is probably not a dependency here?

Reply:
Fixed in commit [`67c6ce2`](https://github.com/ros-industrial/rvip/pull/1/commits/67c6ce25c26b62f1b0602b68c0d5eda50c5a9c8d).